### PR TITLE
Layout testing

### LINKED ISSUE #16: layout-testing

### PR DESCRIPTION
**Pull request by katie210**
Basic grey layout:
- Integrated with master branch changes
- Flexbox 'holy grail' layout
- Basic font scaling (media breakpoints @ 600px, 800px)

Open for extension
closes #16

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -6,39 +6,49 @@
...
-  <main>
+<body ng-controller='ListController as listC' class="wrapper">
+
+  <header class="header">
     
     <section id='search_box'>
-      <label> Search : </label>
-      <input id='search' placeholder='search for makers' ng-model='query' autofocus></input>
+      <label> Search:</label>
+      <input id='search' placeholder='Name, Github username, location...' ng-model='query' autofocus></input>
```
Comment (MarcoCode) [inline app/index.html]: Placeholder not scaling properly on chrome when not in full screen.

**Comment by MarcoCode** [comment]
:shipit:

**Comment by sjmog** [comment]
I like the component focus of the CSS here @katie210 . If you want to challenge yourself even more, rewrite the CSS using [BEM](http://getbem.com/)

File: app/index.html (hunk 1 above)
Comment (katie210) [inline app/index.html]: Tweaked the font-sizing. Scaling issue should be fixed in commit 8de2193

**Pull request merged**